spec/features/content_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "Content Item Details", type: :feature do
```

Comment:
Wouldn't testing the title make it the same as a view test, rather than an integration test?
I was trying not to repeat the `title` test in `show.html.erb_spec.rb`.